db/migrate/20200210223302_add_default_to_pets_description.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultToPetsDescription < ActiveRecord::Migration[5.1]
+  def change
+    change_column :pets, :description, :string, :default => "Looking for a home"
+    Pet.all.where(description: nil).update_all(description: "Looking for a home")
+  end
+end
```

Comment:
This is the migration for giving description a default value. It also goes through the records and updates this wherever that value is nil.

Reply:
I like this addition. Very clean and is sensible.